src/Stratis.Bitcoin.Features.BlockStore/Models/SearchByHashRequest.cs

```diff
@@ -11,5 +11,10 @@ public class SearchByHashRequest : RequestBase
     {
         [Required(AllowEmptyStrings = false)]
         public string Hash { get; set; }
+
+        /// <summary>
+        /// Indicates if the Transactions should be returned with all details, or simply be returned as an string[] with hashes (txids).
+        /// </summary>
+        public bool Verbose { get; set; }
```

Comment:
Renamed to `ShowTransactionDetails` - as mentioned in my comment above.

Reply:
Also when you call the endpoint there is an option to set `OutputJson` to true or false.  Whatever this value is the `Verbose` logic is not considered.

Maybe add a remake around this under the summary - something along the lines of : 

`/// <remarks>This is not considered when <see cref="RequestBase.OutputJson"/> is set to false.</remarks>`

Reply:
Thanks for great feedback, improvements has now been submitted.